### PR TITLE
Help Center: New container

### PR DESCRIPTION
**Pull request by escapemanuele**
#### Changes proposed in this Pull Request

Add:
- a draggable/minimizable container for desktop
- a full screen and minimizable container for mobile

A new `help-center` package has been created.

#### Testing instructions

- checkout the branch
- run `yarn workspace @automattic/help-center run storybook` 
- test the new container in desktop and mobile

### CODE REVIEW

File: packages/help-center/src/components/help-center-container.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+	);
+
+	return (
+		<Draggable disabled={ isMinimized }>
+			<Card className={ classNames } { ...animationProps }>
+				{ containerContent }
+			</Card>
+		</Draggable>
+	);
+};
```
Comment (alshakero) [inline packages/help-center/src/components/help-center-container.tsx]: It seems to be still draggable on mobile. This seems to fix it
```suggestion

	if ( isMobile ) {
		return (
			<Card className={ classNames } { ...animationProps }>
				{ containerContent }
			</Card>
		);
	}
	return (
		<Draggable disabled={ isMinimized }>
			<Card className={ classNames } { ...animationProps }>
				{ containerContent }
			</Card>
		</Draggable>
	);
};
```

File: packages/help-center/src/icons/minimize.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+		<Path
+			fillRule="evenodd"
+			clipRule="evenodd"
+			d="M18.514 9.988l-3.476.016c.627-.626 1.225-1.22 1.82-1.811l.03-.03v-.001c.977-.971 1.944-1.933 3.015-3.004l-1.06-1.06c-1.07 1.069-2.037 2.03-3.013 3.001l-.03.03-1.818 1.809.03-3.449-1.5-.013-.045 5.28-.007.76.76-.004 5.301-.024-.007-1.5zM5.486 14.012l3.477-.016-1.82 1.811-.03.03c-.977.972-1.945 1.934-3.015 3.005l1.06 1.06c1.07-1.068 2.035-2.03 3.012-3V16.9l.03-.03 1.818-1.809-.03 3.449 1.5.013.046-5.28.006-.76-.76.004-5.3.024.006 1.5z"
+			fill="#fff"
+		/>
+	</SVG>
+);
+
+export default minimize;
```
Comment (alshakero) [inline packages/help-center/src/icons/minimize.tsx]: Doesn't seem used.

File: packages/help-center/src/icons/maximize.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+	<SVG width="24" height="24" fill="none" xmlns="http://www.w3.org/2000/svg">
+		<Path
+			fillRule="evenodd"
+			clipRule="evenodd"
+			d="M14.086 5.412l3.476-.015c-.627.625-1.225 1.22-1.82 1.81l-.03.031c-.977.971-1.944 1.934-3.015 3.004l1.06 1.061c1.07-1.07 2.036-2.03 3.013-3.002l.03-.03 1.817-1.808-.03 3.448 1.5.013.046-5.28.007-.759-.76.003-5.301.024.007 1.5zM9.914 18.587l-3.476.016c.627-.625 1.225-1.22 1.82-1.81l.03-.031c.977-.971 1.944-1.934 3.015-3.004l-1.06-1.061c-1.07 1.069-2.036 2.03-3.012 3.001l-.001.001-.03.03-1.817 1.808.03-3.448-1.5-.013-.046 5.279-.007.76.76-.003 5.301-.024-.007-1.5z"
+		/>
+	</SVG>
+);
+
+export default minimize;
```
Comment (alshakero) [inline packages/help-center/src/icons/maximize.tsx]: Doesn't seem used.

File: packages/help-center/src/icons/index.ts [hunk 4]
```diff
@@ -0,0 +1,2 @@
+export { default as maximize } from './maximize';
+export { default as minimize } from './minimize';
```
Comment (alshakero) [inline packages/help-center/src/icons/index.ts]: Doesn't seem used.

File: packages/help-center/src/styles.scss [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+		position: fixed;
+		right: 16px;
+		top: 74px;
+		width: 20vw;
+		z-index: 9999;
+		transition: 0.5s ease;
+
+		.help-center__container-header {
+			cursor: pointer;
+			height: 50px;
```
Comment (alshakero) [inline packages/help-center/src/styles.scss:20]: The header needs less horizontal padding, I suggest the following.
```suggestion
			height: 50px;
			padding-left: 10px;
			// the x button has some white space on the right side, this compensates for that
			padding-right: 5px;
```

File: packages/help-center/src/styles.scss [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+	}
+	@keyframes pullup {
+		0% {
+			transform: translateY( 100vh );
+		}
+
+		100% {
+			transform: translateY( 0 );
+		}
+	}
```
Comment (alshakero) [inline packages/help-center/src/styles.scss]: I think we should only use `opacity` as shown in video pbAok1-2ws-p2. Movement is indicative and can mean something we don't intend to convey.

**Review by alshakero — CHANGES_REQUESTED** [review]
Great. Very close to the finish line!

File: packages/help-center/src/components/help-center-container.tsx (hunk 1 above)
Comment (arcangelini) [reply packages/help-center/src/components/help-center-container.tsx]: Oops, sorry I adjusted that while I was testing and never put it back. @escapemanuele had it working perfectly.

File: packages/help-center/src/styles.scss (hunk 6 above)
Comment (escapemanuele) [reply packages/help-center/src/styles.scss]: Thanks @alshakero, I addressed the issues in the last commit.

**Comment by alshakero** [comment]
We just need to address the tests and this should be good to go!

**Review by alshakero — APPROVED** [review]
Perfect. LGTM.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7216533

Hi @escapemanuele, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include all of the following strings:
- `Maximize Help Center`
- `Minimize Help Center`
- `Close Help Center`

Thank you in advance!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7216533).

**Pull request merged**